test/test_cpp_api_parity.py

```diff
@@ -1,687 +1,64 @@
+MESSAGE_HOW_TO_FIX_CPP_PARITY_TEST_FAILURE = '''
+What should I do when C++ API parity test is failing?
+
+- If you are changing the implementation of an existing `torch.nn` module / functional:
```

Comment:
nit: Instead of functional, "`torch.nn.functional` function" or "`torch.nn.functional` api" is clearer